management/src/test/resources/clusteredConfiguration.txt

```diff
@@ -0,0 +1,24 @@
+caches:
+    cache-1:
+        keyType: java.lang.String
+        valueType: java.lang.String
+        serviceConfigurations:
+        evictionAdvisor:
```

Comment:
Why is `evictionAdvisor` empty? Does that mean it's null? In that case, shouldn't it be left out?